extensions/amp-story/0.1/amp-story.js

```diff
@@ -891,6 +911,22 @@ export class AmpStory extends AMP.BaseElement {
         renderSimpleTemplate(this.win.document, LANDSCAPE_ORIENTATION_WARNING),
         this.element.firstChild);
   }
+
+
+  /**
+   * Build overlay for Landscape mode mobile
+   */
+  buildUnsupportedBrowserOverlay_() {
```

Comment:
Should we also detach the ``amp-story-page`` components from the DOM to make sure there's no sound / video playing?

Also, not very important since we're not playing the story, but as a good practice I think we should always interact with the DOM from vsync tasks.

Reply:
Done, for vsync.

Removing the elements from the DOM causes all kinds of errors since there are a number of race conditions introduced by doing that.  Every AMP element manages its own lifecycle, so the other elements get messed up by being removed from the DOM when they weren't expecting it.